frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-list/experiment-list.component.ts

```diff
@@ -186,6 +187,18 @@ export class ExperimentListComponent implements OnInit, OnDestroy, AfterViewInit
     });
   }
 
+  openExportAllExperimentsDialog() {
+    console.log(this.allExperiments.data);
+    const dialogRef = this.dialog.open(ExportModalComponent, {
+      panelClass: 'export-modal',
+      data: { experiment: this.allExperiments.data }
+    });
+
+    dialogRef.afterClosed().subscribe(result => {
```

Comment:
please remove unused code, I know we have this in other places but I don't think we need to have things we're not using.

Reply:
Removed this part